aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/AWSCognitoAuthPlugin.kt

```diff
@@ -73,6 +73,11 @@ class AWSCognitoAuthPlugin : AuthPlugin<AWSCognitoAuthServiceBehavior>() {
     @VisibleForTesting
     internal lateinit var realPlugin: RealAWSCognitoAuthPlugin
 
+    override fun initialize(context: Context) {
+        super.initialize(context)
```

Comment:
Probably shouldn't have super. This doesn't do anything, but I don't think super is intended to be used here.